compiler/ballerina-treegen/src/main/resources/syntax_tree_descriptor.json

```diff
@@ -1526,6 +1526,29 @@
                 }
             ]
         },
+        {
+            "name": "ParameterizedTypeDescriptorNode",
+            "base": "Node",
```

Comment:
base should be `TypeDescriptorNode`... Its a newly added one with the tree-refactor

Reply:
Changed